raiwidgets/tests/README.md

```diff
@@ -0,0 +1,24 @@
+For running unit tests for `raiwidgets` locally with your changes, you would need an editable install of `raiwidgets`. From the top level directory you can install an editable version of `raiwidgets` using the following command:-
+
+```
+pip install -e raiwidgets
```

Comment:
This seems to be missing the TS part with yarn buildall.

Reply:
These are only python tests. Do we need yarn buildall in this case as well?

Reply:
The tests start the dashboard which won't work if the TS build hasn't happened. The `widget` folder doesn't exist until after the build and it'll fail because it can't find it.

Reply:
Added the bit about `yarn buildall`